controllers/kepler_controller.go

```diff
@@ -444,7 +440,10 @@ func (r *collectorReconciler) ensureDaemonSet(l klog.Logger) (bool, error) {
 				Name:          "http",
 			}},
 		}}
+		// Check if Estimator Patch is required
+		if r.Instance.Spec.Estimator != nil {
```

Comment:
if possible, please double check lint issue.

Reply:
Right. I believe this will be needed for including the estimator patch to kepler's daemonset. I will remove this from this PR because @piparul will be implementing the estimator patch and this is incomplete.